docs/ext/files.rst

```diff
@@ -0,0 +1,47 @@
+.. _ext-files:
+
+************
+Mopidy-Files
+************
+
+Mopidy-Files is an extension for playing music from your local music archive.
+It is bundled with Mopidy and enabled by default. 
+It allows you to browse through your local file system.
+Only files that are considered playable will be shown.
+
+This backend handles URIs starting with ``file:``.
+
+
+Configuration
+=============
+
+See :ref:`config` for general help on configuring Mopidy.
+
+.. literalinclude:: ../../mopidy/files/ext.conf
+    :language: ini
+
+.. confval:: files/enabled
+
+    If the files extension should be enabled or not.
+
+.. confval:: files/media_dir
+
+    A list of directories to be browsable.
+    Optionally the path can be followed by | and a name that will be shown for that path.
```

Comment:
Make the pipe char monospace by adding double backticks around it:

    ... by ``|`` and ...